filer/models/abstract.py

```diff
@@ -38,6 +42,8 @@ class BaseImage(File):
 
     subject_location = models.CharField(_('subject location'), max_length=64, blank=True,
                                         default='')
+    if DJANGO_GTE_19:
+        file_ptr = models.OneToOneField(to='filer.File', related_name='+')
```

Comment:
@mkoistinen any specific reason for not using ``related_name='%(app_label)s_%(class)s_file'`` here?